### ISSUE TITLE
Typo in  "Supporting documents" section

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wcag#3937]
Ref:
https://www.w3.org/TR/WCAG22/#wcag-2-2-supporting-documents  List item 1. “This includes content from WCAG 2.0, 2.1 2.2 [...]“. There’s an “and” missing.

**Comment by mbgower** [comment]
The change for F26 was merged several weeks ago, and seems irrelevant to this change.
https://github.com/w3c/wcag/pull/3863

**Comment by daniel-montalvo** [comment]
Thanks @mbgower 

It's now updated.

**Comment by ljoakley** [comment]
https://www.w3.org/TR/WCAG22/#wcag-2-2-supporting-documents List item 1. “This includes content from WCAG 2.0, 2.1 2.2 [...]“. There’s an “and” missing. The line should read "..This includes content from WCAG 2.0, 2.1 and 2.2"
